cmd/egress/cmd.go

```diff
@@ -4,98 +4,77 @@ import (
 	"context"
 	"fmt"
 	"os"
-	"time"
 
-	"github.com/aws/aws-sdk-go-v2/credentials"
 	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
 	"github.com/openshift/osd-network-verifier/pkg/cloudclient"
+	"github.com/openshift/osd-network-verifier/pkg/parameters"
 	"github.com/spf13/cobra"
 )
 
-var (
-	defaultTags            = map[string]string{"osd-network-verifier": "owned", "red-hat-managed": "true", "Name": "osd-network-verifier"}
-	regionEnvVarStr string = "AWS_REGION"
-	regionDefault   string = "us-east-2"
-)
-
-type egressConfig struct {
-	vpcSubnetID  string
-	cloudImageID string
-	instanceType string
-	cloudTags    map[string]string
-	debug        bool
-	region       string
-	timeout      time.Duration
-	kmsKeyID     string
-}
+var config = cloudclient.CmdOptions{}
```

Comment:
This should be a locally defined struct specifically for this cli command.

Reply:
Thanks for starting review! As per our discussion, the cmd cli (which is an example of downstream codes) may define their variables any way they want, as long as they pas them in as struct defined on our cloudclient package. As Benson pointed out during the meet, how it is defined is not up to us as long as it is passed in as per prescribed "egressValidate" struct. 

But it doesn't hurt to use a struct, so I'll update the cli to use a struct instead of variable.